__tests__/app.test.js

```diff
@@ -47,4 +47,37 @@ describe("GET /api", () => {
     });
 });
 
-
+describe("/api/articles/:article_id", () => {
+    it("GET 200: returns a single article object when given a valid and existing article id", () => {
+        return request(app)
+        .get("/api/articles/5")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article.article_id).toBe(5) 
+            expect(typeof body.article.article_id).toBe("number")
+            expect(typeof body.article.author).toBe("string")
+            expect(typeof body.article.title).toBe("string")
+            expect(typeof body.article.body).toBe("string")
+            expect(typeof body.article.topic).toBe("string")
+            expect(typeof body.article.created_at).toBe("string")
+            expect(typeof body.article.votes).toBe("number")
+            expect(typeof body.article.article_img_url).toBe("string")
```

Comment:
You might want to look at the jest matchers, `toHavePropterty` and `expect.any(<type>)` in the jest docs